packages/x/x-markdown-css.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "x-markdown-css",
+  "filename": "x-markdown-css.min.css",
+  "description": "A simple and customizable markdown CSS stylesheet for everyone.",
+  "license": "MIT",
+  "keywords": [
+    "markdown",
+    "css",
+    "sass",
+    "scss",
+    "coffeescript",
+    "grunt",
+    "javascript",
+    "postcss",
+    "prose",
+    "blog",
+    "article",
+    "documentation"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kuohuanhuan/x-markdown-css.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "x-markdown-css",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "*.css",
+          "*.css.map"
```

```suggestion
          "*.css?(.map)"
```
This can be reduced to a single line